docs/data-sources/all_namespaces.md

```diff
@@ -14,6 +14,9 @@ In Kubernetes, namespaces provide a scope for names and are intended as a way to
 ## Example Usage
 
 ```terraform
+# Copyright (c) HashiCorp, Inc.
```

Comment:
Not sure how all these copyright comments got added but they shouldn't be in the example snippets?

Reply:
Right, Not sure if that's intended behavior - I'll raise it with the devex team but basically when the files are migrated to the new structure, it picks up everything from the `.tf` file including the header.